service/key/key.go

```diff
@@ -88,3 +89,23 @@ func WorkerInstanceType(customObject awstpr.CustomObject) string {
 
 	return instanceType
 }
+
+func ToCustomObject(v interface{}) (awstpr.CustomObject, error) {
+	if v == nil {
+		return awstpr.CustomObject{}, microerror.Maskf(wrongTypeError, "expected '%T', got '%T'", &awstpr.CustomObject{}, v)
+	}
+
+	customObjectPointer, ok := v.(*awstpr.CustomObject)
+	if !ok {
+		return awstpr.CustomObject{}, microerror.Maskf(wrongTypeError, "expected '%T', got '%T'", &awstpr.CustomObject{}, v)
+	}
+	customObject := *customObjectPointer
+
+	return customObject, nil
+}
+
+func MainStackName(customObject awstpr.CustomObject) string {
```

Comment:
FYI we tried to order methods alphabetically in the past.

Reply:
Sure, done